### ISSUE TITLE
Warnings generated by the standard library

### ISSUE THREAD

**Issue by Zimmi48** [coq/coq#6594]
#### Version

8.8+alpha

#### Description of the problem

The compilation generates the following warnings:

```
Warning: Summary was captured out of module scope for entry field-tac-carrier-table
Warning: Summary was captured out of module scope for entry field-tac-name-table
Warning: Summary was captured out of module scope for entry ring-tac-carrier-table
Warning: Summary was captured out of module scope for entry ring-tac-name-table
```

According to @ejgallego, it should be made into a debug warning and disabled by default.

**Comment by ppedrot** [comment]
What @ejgallego proposes is independent from this particular problem. I'm fairly sure this is due to field and ring messing with the libobject assumptions here.

**Comment by ejgallego** [comment]
@ppedrot I thing this happens because the .v files require a plugin inside a section.

So I'd rather leave the warning there until the beta, and fix the .v files. Indeed, after a second thought it may be not so bad to leave the warning there but mentioning that requiring ml plugins inside sections is ... well, I don't really know how bad it can be.

**Comment by Zimmi48** [comment]
Indeed, if the warning is kept the message could be improved :P

**Comment by ejgallego** [comment]
So I propose two separate action paths then:
- fix the stdlib so it doesn't load ML plugins inside sections.
- clarify the message and turn it into a proper warning.

**Comment by maximedenes** [comment]
Will be fixed by #6852 and #6934.
